fault.h

```diff
@@ -24,6 +24,20 @@
  */
 #define KIBOSH_FAULT_TYPE_STR_LEN 32
 
+/**
+ * Constant flags for byte corruption modes.
+ */
+#define CORRUPT_ZERO 1000
+#define CORRUPT_RAND 1001
+#define CORRUPT_ZERO_SEQ 1100
+#define CORRUPT_RAND_SEQ 1101
+#define CORRUPT_DROP 1200       // this will silently drop data from write calls
```

Comment:
What is the reason for picking numbers like these? Could this just be an enum?

Reply:
Referring to the previous comment, I picked large numbers to distinguish corruption faults from unreadable/unwritable faults. An enum would work as well.